File: 01.ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Lab/src/P11_OddNumber.java
import java.util.Scanner;
public class P11_OddNumber {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);

        for(int i = 1; i <= 10; i++) {
            int number = Integer.parseInt(console.nextLine());
            if(number % 2 == 0) {
                System.out.println("Please write an odd number.");
            } else {
                System.out.println("The number is: " + Math.abs(number));
                break;
            }
        }
    }
}

File: 01.ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Lab/P11_OddNumber/P11_OddNumber.cs
using System;

namespace P11_OddNumber
{
    class P11_OddNumber
    {
        static void Main(string[] args)
        {   
            for (int i = 1; i <= 10; i++)
            {
                int number = int.Parse(Console.ReadLine());
                if (number % 2 == 0)
                {
                    Console.WriteLine("Please write an odd number.");
                }
                else
                {
                    Console.WriteLine("The number is: " + Math.Abs(number));
                    break;
                }
            }
        }
    }
}
